src/webfiles/portal/js/juxt.js

```diff
@@ -36,13 +37,49 @@ function initYeah() {
             parent.classList.remove('yeah');
             if(count) count.innerText -= 1;
             wiiuSound.playSoundByName('SE_OLV_MII_CANCEL', 1);
+            if (document.querySelector('#post-' + id + ' .yeah-list')) {
```

Comment:
Does the Wii U/3DS support template literals? This is a somewhat modern feature, being introduced in Chrome 42, so I'm not sure. If they do, I would prefer those over string concatenation. If not, this is fine

Reply:
It seems it does support it, as i tested with it on TVii too